src/pyfmi/master.pyx

```diff
@@ -782,7 +782,7 @@ cdef class Master:
             index_end = index_start + self.models_dict[model]["local_output_len"]
             local_output_vref_array = (<FMUModelCS2>model).get_real(self.models_dict[model]["local_output_vref_array"])
             for i, index in enumerate(range(index_start, index_end)):
-                y[index] = local_output_vref_array[i]
+                y[index] = local_output_vref_array[i].item()
```

Comment:
It would be an array of length 1, e.g., `[1.23]` and this type of assignment gave a warning from numpy.

`np.ndarray.item() (== np.ndarray.item(None))` converts this to a scalar and gives an Exception if it is not a scalar. I strongly prefer this approach over `y[index] = local_output_vref_array[i][0]`.

Reply:
I see, thanks!